reV/supply_curve/sc_aggregation.py

```diff
@@ -281,8 +285,42 @@ def __init__(self, excl_fpath, tm_dset, econ_fpath=None,
             ``layer_dset_name`` is a dataset in the exclusion h5 file
             and the ``kwarg: value`` pair is a keyword argument to
             the :class:`reV.supply_curve.exclusions.LayerMask` class.
-            If ``None`` or empty dictionary, no exclusions are applied.
-            By default, ``None``.
+            For example::
+
+                excl_dict = {
+                    "typical_exclusion": {
+                        "exclude_values": 1,
```

Comment:
I think this docstring pops up one more place so comment is general... Can we have this not be 0 or 1 to avoid confusion with a boolean option? I also have a comment on the confusion of a boolean below.

Reply:
Yes, great point. Updated in both places.